micronota/bfillings/_base.py

```diff
@@ -0,0 +1,83 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015--, micronota development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from abc import ABCMeta, abstractmethod
+
+from skbio import Sequence
+
+
+class IntervalMetadataPred(metaclass=ABCMeta):
```

Comment:
yeah, one is for structual annotation and the other function annotation.
what do u mean by straightup skbio.sequence?

On Mon, Mar 7, 2016 at 2:39 PM, Jamie Morton <notifications@github.com>
wrote:

> In micronota/bfillings/_base.py
> <https://github.com/biocore/micronota/pull/54#discussion_r55285993>:
>
> > @@ -0,0 +1,83 @@
> > +# ----------------------------------------------------------------------------
> > +# Copyright (c) 2015--, micronota development team.
> > +#
> > +# Distributed under the terms of the Modified BSD License.
> > +#
> > +# The full license is in the file COPYING.txt, distributed with this software.
> > +# ----------------------------------------------------------------------------
> > +
> > +from abc import ABCMeta, abstractmethod
> > +
> > +from skbio import Sequence
> > +
> > +
> > +class IntervalMetadataPred(metaclass=ABCMeta):
>
> Do we need two classes? That is already taken care of with straight-up
> skbio.Sequences.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/micronota/pull/54/files#r55285993>.
>